libr/asm/p/asm_ppc_as.c

```diff
@@ -0,0 +1,132 @@
+/* radare - LGPL - Copyright 2015-2020 pancake */
```

Comment:
The copyright is wrong. This file wasnt born in 2015 and it was not commited by me. Put your name